pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -81,7 +81,7 @@ def create_row(device_data, selected, mutable):
         vendor=device_data.attrs.get("vendor", ""),
         interconnect=device_data.attrs.get("bus", ""),
         serial=device_data.attrs.get("serial", ""),
-        wwid=device_data.attrs.get("path-id", ""),
+        wwid=device_data.attrs.get("path-id", device_data.attrs.get("wwn", "")),
```

Comment:
I would prefer to use `device_data.attrs.get("path-id", "") or device_data.attrs.get("wwn", "")` in case that `path-id` is present, but empty (it shouldn't happen, but just in case).

Reply:
Changed with [087d280](https://github.com/rhinstaller/anaconda/pull/5249/commits/087d280018f2dc32e1264fd4f6546c59fad5f80a) and tested successfully: For multipath devices the WWID column shows the wwid/wwn, and for single path (zfcp-attached) SCSI disks the WWID column shows path-id just as before the change.